README.md

```diff
@@ -303,6 +303,93 @@ The credit card charge result may contains `redirect_url` for 3DS authentication
 For full example on Credit Card 3DS transaction refer to:
 - [Flask App examples](/examples/flask_app) that implement Snap & Core Api
 
+### 2.2.D Subscription API
+You can see some Subscription API examples [here](examples/subscription), [Subscription API Docs](https://api-docs.midtrans.com/#subscription-api)
+
+```python
+# Create Subscription API instance
+subscription = midtransclient.CoreApi(
+    is_production=False,
+    server_key='YOUR_SERVER_KEY',
+    client_key='YOUR_CLIENT_KEY'
+)
+# Build parameter
+param = {
+    "name": "SUBSCRIPTION-STARTER-1",
+    "amount": "100000",
+    "currency": "IDR",
+    "payment_type": "credit_card",
+    "token": "436502qFfqfAQKScMtPRPdZDOaeg7199",
+    "schedule": {
+      "interval": 1,
+      "interval_unit": "month",
+      "max_interval": 3,
+      "start_time": "2021-10-01 07:25:01 +0700"
+    },
+    "metadata": {
+      "description": "Recurring payment for STARTER 1"
+    },
+    "customer_details": {
+      "first_name": "John A",
+      "last_name": "Doe A",
+      "email": "johndoe@email.com",
+      "phone": "+62812345678"
+    }
+}
+create_subscription_response = subscription.create_subscription(param)
+
+subscription_id_response = create_subscription_response['id']
+# get subscription by subscription_id
+get_subscription = subscription.get_subscription(subscription_id_response)
+
+# disable subscription by subscription_id
+disable_subscription = subscription.disable_subscription(subscription_id_response)
+
+# enable subscription by subscription_id
+enable_subscription = subscription.enable_subscription(subscription_id_response)
+
+# update subscription by subscription_id
+update_param = {
+    "name": "SUBSCRIPTION-STARTER-1-UPDATE",
+    "amount": "100000",
+    "currency": "IDR",
+    "token": "436502qFfqfAQKScMtPRPdZDOaeg7199",
+    "schedule": {
+      "interval": 1
+}
+update_subscription = subscription.update_subscription(subscription_id_response, update_param)
+```
+
+### 2.2.E Tokenization API
+You can see some Tokenization API examples [here](examples/tokenization), [Tokenization API Docs](https://api-docs.midtrans.com/#gopay-tokenization)
+
+```python
+# Create Tokenization API instance
+tokenization = midtransclient.CoreApi(
+    is_production=False,
+    server_key='YOUR_SERVER_KEY',
+    client_key='YOUR_CLIENT_KEY'
+)
+# Build parameter
+param = {
+  "payment_type": "gopay",
+  "gopay_partner": {
+    "phone_number": "81234567891",
+    "country_code": "62",
+    "redirect_url": "https://midtrans.com"
```

Comment:
don't use midtrans.com as example, it will confuse merchant and make them think it must use midtrans website. Use something like `https://mywebstore.com/gopay-linking-finish`